packages/grid_client/tests/modules/applications/casperlabs.test.ts

```diff
@@ -0,0 +1,267 @@
+import axios from "axios";
+import { setTimeout } from "timers/promises";
+
+import { FilterOptions, GatewayNameModel, generateString, GridClient, MachinesModel, randomChoice } from "../../../src";
+import { config, getClient } from "../../client_loader";
+import { bytesToGB, generateInt, getOnlineNode, log, splitIP } from "../../utils";
+
+jest.setTimeout(1500000);
```

Comment:
Why the timeout?

Reply:
The gateway takes time to load and get the site ready, more than 10 minutes in some cases.

Reply:
But we did set a timeout before checking for the gateway, right?

Reply:
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/109473873/6c399923-d59d-40a9-b664-55e7e299d354)